src/codelens/TestCodeLensProvider.ts

```diff
@@ -21,6 +22,10 @@ export class TestCodeLensProvider implements CodeLensProvider, Disposable {
     }
 
     public async provideCodeLenses(document: TextDocument, _token: CancellationToken): Promise<CodeLens[]> {
+        if (!testFileWatcher.isOnTestSourcePath(document.uri.fsPath)) {
```

Comment:
About filter the codelens, it seems vscode supports glob pattern on whether to trigger codelens.

```typescript
	export interface DocumentFilter {

		/**
		 * A language id, like `typescript`.
		 */
		language?: string;

		/**
		 * A Uri [scheme](#Uri.scheme), like `file` or `untitled`.
		 */
		scheme?: string;

		/**
		 * A [glob pattern](#GlobPattern) that is matched on the absolute path of the document. Use a [relative pattern](#RelativePattern)
		 * to filter documents to a [workspace folder](#WorkspaceFolder).
		 */
		pattern?: GlobPattern;
	}

```